src/app/_component/HomePage.tsx

```diff
@@ -0,0 +1,5 @@
+interface Props {}
```

Comment:
Does _component have significance?

Reply:
I store the components that are used only 1 time specifically for the page. Like the HomePage would appear only on the home page and not other pages.

Reply:
This is an unexpected convention (i.e. We did not have agreement on this)

However, I think I like it.